tests/docker/install/_helper_check_wheel.sh

```diff
@@ -101,7 +91,7 @@ function get_wheel(){
             exit 1
         fi
     else
-        pip_options=(--only-binary solarwinds-apm --dest "$wheel_dir")
+        pip_options=(--only-binary solarwinds-apm --dest "$wheel_dir" solarwinds-apm)
```

Comment:
Minor is the addition of `solarwinds-apm` needed? Seeing later that we do seem to append it to pip_options.

Reply:
I think it is, technically. pip `--only-binary` and `--no-binary` are supposed to have a `<format_control>` value provided: https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-only-binary

The way it's currently written in the install tests still works and I'm not sure why given the above. The install tests pass on GH (have been for a long time) and this also passes on my local on `main` branch: `MODE=testpypi docker-compose run --rm py3.7-install-alpine3.12`.

The missing `<format_control>` becomes an issue when I try to install from the testbed for `flask-from-testpypi`. Both of these error:

```
pip install --no-binary solarwinds-apm==0.8.0
pip install --no-binary solarwinds-apm
```
But this is ok:
```
pip install --no-binary solarwinds-apm==0.8.0 solarwinds-apm
```
Actually this is failing too...
```
pip install --no-binary solarwinds-apm solarwinds-apm
```

...Ok. I'm going to revert [b7fcfd6](https://github.com/solarwindscloud/solarwinds-apm-python/pull/119/commits/b7fcfd66c1874c41f2436069975b5dfed7ef57ce) and do a fresh publish and test.